src/app/app.html

```diff
@@ -1,14 +1,17 @@
 <ion-split-pane>
   <ion-menu [content]="content">
-    <ion-header aria-hidden="true">
+    <ion-header>
       <ion-toolbar color="primary">
-        <ion-title>PVTrAck</ion-title>
+        <ion-title>
+          PVTrAck
```

Comment:
Hmm. For some reason I can't hide it agian and have the menuClose button as `aria-hidden="false"`. It won't read the menuClose button. But I need it to be on the same line as PVTrAck